src/entitySearchResultsList.js

```diff
@@ -98,7 +102,27 @@ export default class EntitySearchResultsList extends LightningElement {
         this.dispatchEvent(new CustomEvent('loadmore', {detail: {name: "juan"}}));
     }
 
-    handleSort(evt){
-        this.dispatchEvent(new CustomEvent('sort', {detail: {name: "juan"}}));
+    handleSort({ detail }) {
+        this.sortedBy = detail.fieldName;
+        this.sortDirection = detail.sortDirection;
+        this.performSort(this.sortedBy, this.sortDirection);
+    }
+
+    performSort(sortColumn, sortDirection) {
```

Comment:
sorting locally.

Reply:
yeah I wanted to bubble up the event in order to send a new ask request with the sorting parameters, something to explore after the hack I suppose